spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -197,5 +208,53 @@
         expect(pdf_fields["ZIP Code"]).to eq "08037"
       end
     end
+
+    describe "filing status" do
+      context "single" do
+        before do
+          submission.data_source.direct_file_data.filing_status = 1
+        end
+        it "checks the appropriate box" do
```

Comment:
[dust/pebble] cam we rename these tests to include the expectation? I like being able to read test names to know what behavior to expect, so when it just says "appropriate box" it means you have to actually read the expect line of the test itself to understand which box is appropriate. Could update to something like "checks the Choice1 box" just to be clearer and more readable